Creates sdcore management interface (#104)
canonical/charm-relation-interfaces: interfaces/sdcore_management/README.md

> # `sdcore_management`

Reviewer: ```suggestion
# `SD-Core Management`
```

## Usage

> Within Charmed-5G, components that makes configuration changes to the network need to have access to the configuration and management service.

Reviewer: ```suggestion
Within Charmed-5G, the components that makes configuration changes to the network require access to the configuration and management service.
```

> The `sdcore_management` relation interface describes the expected behavior of any charm claiming to be able to provide or consume the information to access the configuration service in SD-Core.

Reviewer: ```suggestion
The `sdcore_management` relation interface describes the expected behavior of any charm claiming to provide or consume the information to access the configuration service in SD-Core.
```
